docs/src/doc/setup/gradle.md

```diff
@@ -55,16 +89,15 @@ class Simple: Spatial() {
 
 More will be explained in the [classes](../user-guide/classes.md) section of user guide, but for now `@RegisterClass` will register
 the annotated class to Godot.
-
-!!! note ""
-    The plugin automatically generates the appropriate `gdns` files which can be found at `src/gdns`. It is up to you whether you want to include those files in source control or not.
     
 Now we can trigger a build.
```

Comment:
IMO `org.gradle.jvmargs=-Xmx3G` inside `gradle.properties` is missing as the first build failed for me because of running out of memory

Reply:
maybe also an explanation why we need more memory would be good in an info box or something like that